### DISCUSSION TITLE
Unique number of subject_ids is much greater than 40,000

### DISCUSSION THREAD

**Discussion by MichalWeisman** [MIT-LCP/mimic-code#1605] [MIMIC-IV]
Hello, 

In the dataset's documentation, it is mentioned that the data contains information for 40,000 patients.
However, when computing the number of unique values in the subject_id column I receive a much greater number.
For example, I would like to know the number of patients who had a blood glucose test:
```
glucose_df = lebevents[lebevents['itemid'] == 50931] # 50931 is the itemid of the glucose test
print(glucose_df['subject_id'].nunique())
```
The output is 247,005 which is much greater than 40,000.
Would appreciate any help in understanding this.

Thanks

**Comment by alistairewj** [comment]
Where in the documentation? That likely needs to be updated.

The correct statement is there are roughly ~40,000 *ICU* patients (more if I recall correctly), but there are around ~250,000 patients who went to either the ICU or the ED. This was a change between MIMIC-III and MIMIC-IV, as we expanded MIMIC-IV to include patients in [MIMIC-IV-ED](https://physionet.org/content/mimic-iv-ed/2.2/).
